content/thirdpartynotices.txt

```diff
@@ -0,0 +1,32 @@
+This file is based on or incorporates material from the projects listed below
+(Third Party IP). The original copyright notice and the license under which
+Microsoft received such Third Party IP, are set forth below. Such licenses and
+notices are provided for informational purposes only. Microsoft licenses the
+Third Party IP to you under the licensing terms for the Microsoft product.
+Microsoft reserves all other rights not expressly granted under this agreement,
+whether by implication, estoppel or otherwise. 
+
+libuv v. 1.7.3
```

Comment:
@Eilon I don't think it makes a difference for us at this point. The @lodejard's fix is in neither version so either you change the license or I change the submodule to point to 1.7.3 instead of 1.7.4